frontend/src/routes/Infrastructure/Automations/AnsibleAutomationsForm.tsx

```diff
@@ -543,6 +543,8 @@ function EditAnsibleJobModal(props: {
             <AcmForm>
                 <AcmForm>
                     <AcmSelect
+                        maxHeight="12em"
```

Comment:
I'm not sure that we need the maximum height here... if the list is long, with the fix it will extend below the modal, making better use of screen space for scrolling through a long list.

Let's check with @imjoyjean though - do you think a dropdown on a modal should try to fit within the modal, or can it extend below as long as it doesn't get hidden inside the modal? ie. compare this img to the video Randy posted:
![image](https://user-images.githubusercontent.com/42188127/129973061-ffd0e9eb-eeda-4434-be02-4d7ec8801e3d.png)

Reply:
Hmm, t seems like if your window is not tall, the dropdown without a maximum height will just get clipped, so we may in fact need to set one.
![image](https://user-images.githubusercontent.com/42188127/129973460-afd3e970-b590-4a67-b466-0f1be997ac1e.png)

Reply:
I think the ideal would be that the max height of the dropdown should be within the borders of the modal, and you can scroll up and down in that fixed height. Let me check with patternfly to see if they have any guidance for it.

Reply:
@Randy424 Joy did some research and is suggesting setting a max height that fits about 10 items. This may overflow the modal, but hopefully not the user's window.